test/cmd/status.sh

```diff
@@ -21,38 +21,38 @@ cp "${KUBECONFIG}" "${login_kubeconfig}"
 
 os::test::junit::declare_suite_start "cmd/status"
 # login and ensure no current projects exist
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
+os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/server-ca.crt' -u test-user2 -p anything"
 os::cmd::expect_success 'oc delete project --all'
 os::cmd::try_until_text "oc get projects -o jsonpath='{.items}'" "^\[\]$"
 os::cmd::expect_success 'oc logout'
 
 # remove self-provisioner role from user and test login prompt before creating any projects
 os::cmd::expect_success "oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated:oauth --kubeconfig='${login_kubeconfig}'"
 
-# login as 'test-user'
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
+# login as 'test-user2'
```

Comment:
Why this change?

Reply:
I wanted to use a different user from the one being used to log in in `test-cmd`. It was mostly a way for me to check that I was using the correct kubeconfig context in this test